math/orpie/Portfile

```diff
@@ -18,17 +18,39 @@ long_description    Orpie is a fullscreen RPN calculator for the console. Its \
                     functionality, command completion, and a visible interactive \
                     stack.
 
-homepage            http://pessimization.com/software/orpie/
-master_sites        ${homepage}
+# not actually used for compiling -- OPAM is used instead
+homepage            https://github.com/pelzlpj/orpie/
+master_sites        ${homepage}/archive/refs/tags/
+distname            release-${version}
+worksrcdir          orpie-release-${version}
 
-checksums           rmd160 61ee7019ec5d141aade27d1c78770c2b11803085 \
-                    sha256 de557fc7f608c6cb1f44a965d3ae07fc6baf2b02a0d7994b89d6a0e0d87d3d6d
+checksums           release-1.6.1.tar.gz \
+                    rmd160  081a2f5302d69a0491e3a7aaaa78cd9483690eec \
+                    sha256  1ae5a99fbbf492869664e4bd81e6bf5cf65bdaf5027908c1e9313843bb5d5749 \
+                    size    277420
+
+# skip fetch step
+fetch {}
+
+# skip configure step
+use_configure no
+
+build {
+    ui_msg "If you haven't already, you need to run 'xcode-select --install'."
+    ui_msg "This will take some time to complete - please be patient."
```

Comment:
This message is not desirable. What does this port require: Xcode, the Xcode command line tools, or either? If Xcode, then `use_xcode yes` should be used. If either, then nothing is needed; it can be assumed that the user has installed one or the other.

Reply:
It seems that libc is required, and this is installed via `xcode-select --install`.  For this case would `use_xcode yes` be needed?